src/frontend/src/flows/manage/recoveryMethodsSection.ts

```diff
@@ -26,19 +26,42 @@ export const recoveryMethodsSection = ({
   addRecoveryPhrase: () => void;
   addRecoveryKey: () => void;
 }): TemplateResult => {
+  const i18n = new I18n();
+
+  const copy = i18n.i18n(copyJson);
+  const wrapClasses = ["l-stack"];
+  const warnNoRecovery = isNullish(recoveryPhrase) && isNullish(recoveryKey);
+
+  if (warnNoRecovery) {
+    wrapClasses.push("c-card", "c-card--narrow", "c-card--warning");
+  }
   return html`
-    <aside class="l-stack">
+    <aside class=${wrapClasses.join(" ")}>
+      ${warnNoRecovery
+        ? html`
+            <span
+              class="c-card__label c-card__label--hasIcon"
+              aria-hidden="true"
+            >
+              <i
+                class="c-card__icon c-icon c-icon--error__flipped c-icon--inline"
+                >${warningIcon}</i
+              >
+              <h2>${copy.security_warning}</h2>
+            </span>
+          `
+        : undefined}
```

Comment:
in the code base of II, didn't you use "" for those use cases?

Reply:
It turns out, the code-base is not consistent. The `authenticatorsSection.ts` uses `undefined`. But let's switch to `""` because it's shorter and more common.

Reply:
Ah ok, good to know.